src/routes/register/registerModel.ts

```diff
@@ -0,0 +1,30 @@
+import * as mongoose from 'mongoose'
+
+const registerSchema = new mongoose.Schema({

```

Comment:
Schema też powinna się nazywać od nazwy zapisywanego modelu

Reply:
zmienione na userSchema

Reply:
zapisany model to `userAccount` a nazwa scheme to `userSchema`. Zdecyduj czy model bazy ma się nazywać `user` czy zmienić nazwę scheme na `userAccountScheme`. Tak jak pisałem schema powinna się nazywać od nazwy modelu czyli na wzór np `{model}Schema`

Reply:
zmienione na userAccountSchema